po/es.po

```diff
@@ -0,0 +1,606 @@
+
+#: assets/ui/media-toc.ui:967
+msgid "mkvmerge text format"
+msgstr "Formato de texto de mkvmerge"
+
+#: assets/ui/media-toc.ui:983
+msgid ""
+"mkvmerge compliant text format.\n"
+"Flat table of contents.\n"
+"Time resolution: 1 ms."
+msgstr ""
+"Formato de texto compatible mkvmerge.\n"
+"Tabla de contenido plana. \n"
+"Unidad de tiempo: 1 ms."
+
+#: assets/ui/media-toc.ui:1017
+msgid "Cue Sheet format"
+msgstr "Formato Cue Sheet"
+
+#: assets/ui/media-toc.ui:1033
+msgid ""
+"Cue Sheet compliant text format.\n"
+"Hierachical table of contents.\n"
+"Time resolution: 1 frame (1/75 s)."
+msgstr ""
+"Formato de texto compatible Cue Sheet.\n"
+"Tabla de contenido jerárquica.\n"
+"Unidad de tiempo: 1 trama (1/75 s)."
+
+#: assets/ui/media-toc.ui:1067
+msgid "Matroska Container"
+msgstr "Contenedor Matroska"
+
+#: assets/ui/media-toc.ui:1083
+msgid ""
+"Binary audio, video & text media container.\n"
+"Hierachical table of contents.\n"
+"Time resolution: 1 ns."
+msgstr ""
+"Contenedor binario de audio, vídeo y texto.\n"
+"Tabla de contenido jerárquica.\n"
+"Unidad de tiempo: 1 ns.\n"
+
+#. Button label
+#: assets/ui/media-toc.ui:1127
+msgid "Export"
+msgstr "Exportar"
+
+#. Button tooltip
+#: assets/ui/media-toc.ui:1131
+msgid "Export media to selected format"
+msgstr "Exportar archivo multimedia al formato seleccionado"
+
+#: assets/ui/media-toc.ui:1149
+msgid "Export Formats"
+msgstr "Formatos de exportación"
+
+#: assets/ui/media-toc.ui:1267
+msgid "Flac"
+msgstr "Flac"
+
+#: assets/ui/media-toc.ui:1283
+msgid ""
+"Audio lossless compression.\n"
+"Tags support."
+msgstr ""
+"Compresión de audio sin pérdidas.\n"
+"Gestión de etiquetas."
+
+#: assets/ui/media-toc.ui:1335
+msgid "Wave / PCM"
+msgstr "Wave / PCM"
+
+#: assets/ui/media-toc.ui:1351
+msgid ""
+"No compression.\n"
+"Tags support."
+msgstr ""
+"Sin compresión.\n"
+"Gestión de etiquetas."
+
+#: assets/ui/media-toc.ui:1403
+msgid "Opus"
+msgstr "Opus"
+
+#: assets/ui/media-toc.ui:1419 assets/ui/media-toc.ui:1487
+#: assets/ui/media-toc.ui:1555
+msgid ""
+"Audio lossy compression.\n"
+"Tags support."
+msgstr ""
+"Compresión de audio con pérdidas.\n"
+"Gestión de etiquetas."
+
+#: assets/ui/media-toc.ui:1471
+msgid "Vorbis"
+msgstr "Vorbis"
+
+#: assets/ui/media-toc.ui:1539
+msgid "MP3"
+msgstr "MP3"
+
+#. Button label
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1598 assets/ui/media-toc.ui:1979
+msgid "Split"
+msgstr "Dividir"
+
+#. Button tooltip
+#: assets/ui/media-toc.ui:1602
+msgid "Split audio stream and encode to selected format"
+msgstr "Dividir la secuencia de audio y codificarla al formato seleccionado"
+
+#: assets/ui/media-toc.ui:1620
+msgid "Split Formats"
+msgstr "Formatos de corte"
+
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1794
+msgid "Display"
+msgstr "Visualización"
+
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1856
+msgid "Streams"
+msgstr "Flujos"
+
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1918
+msgid "Export\t"
```

Comment:
Thanks for reporting this in PM. I updated the UI definition in https://github.com/fengalin/media-toc/commit/d56014e964d606a5e7aef910c3b8df2946feb7d9, so it should be fixed once you `git pull --rebase` and `po/update es`.

Reply:
done!